dapps/profile.vue/src/components/root/root.ts

```diff
@@ -108,8 +108,8 @@ export default class ProfileRootComponent extends mixins(EvanComponent) {
    * Load currents profiles data.
    */
   async setupProfile(): Promise<void> {
-    const { address } = this.$route.params;
     const runtime = this.getRuntime();
+    const address = this.$route.params.address || runtime.activeIdentity;
```

Comment:
In which situations is the address missing?

Reply:
It's for the wallet... This one is currently excluded from the second level profile navigation and is opened via the dapp-wrapper, always in the context of the current user. Maybe we need to exclude the wallet later to a seperated DApp.